src/sonic-config-engine/minigraph.py

```diff
@@ -261,17 +261,15 @@ def parse_cpg(cpg, hname):
                 end_router = session.find(str(QName(ns, "EndRouter"))).text
                 end_peer = session.find(str(QName(ns, "EndPeer"))).text
                 if end_router == hname:
-                    bgp_sessions.append({
+                    bgp_sessions[start_peer] = {
                         'name': start_router,
-                        'addr': start_peer,
                         'peer_addr': end_peer
-                    })
```

Comment:
here, should we change the 'peer_addr' -> 'addr', and 'name' -> 'peer_name'?

this is because, the end_router is myself, and the name is really the my peer name.